proposals/004-skips/README.md

```diff
@@ -0,0 +1,179 @@
+# SKIP 000 -Sidecar Containers in Spin Operator
+
+Summary: This proposal introduces sidecar container support in the Spin Operator, enabling Spin applications to run alongside Linux containers within the same pod. This allows for added functionality such as monitoring, logging, and security.
+
+Owner: jiazh@microsoft.com
+
+Impacted Projects:
+
+- [x] spin-operator
+- [ ] `spin kube` plugin
+- [ ] runtime-class-manager
+- [ ] containerd-shim-spin
+- [ ] Governance
+- [ ] Creates a new project
+
+Created: October 16th, 2024
+
+Updated: October 16th, 2024
+
+## Background
+
+The [Runwasi] project supports running Wasm workloads in the same pod as Linux containers. Currently, the SpinApp Custom Resource Definition (CRD) does not support running [Sidecar Containers]. Sidecars are important in use cases where additional services need to run in the same pod as the Spin application, enabling low-latency communication and resource sharing.
+
+[Sidecar Containers]: (https://kubernetes.io/docs/concepts/workloads/pods/sidecar-containers/)
```

Comment:
This Kubernetes feature was new to me.  I was thinking about the sidecar pattern and didn't realize it was an actual feature.  In the description above could you add sentence on the difference and why Sidecar feature (capital S) was chosen?